### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Amazinggracee**
### Project requirements

- [ ] Add any missing JSX elements to your project (without adding extra functionality)
- [ ] Your final design should match Zeplin's design (this includes all elements + the same styling)
- [ ] You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -307,16 +329,30 @@
...
+.comments button {
+  border: none;
+  background: transparent;
+  color: var(--blue0);
+}
+
+.comments button:hover {
+  border: 0.01rem solid var(--blue0);
+  background: transparent;
+}
```
Comment (Davidosky007) [inline src/App.css:338]: * Well done with the design but there are still some differences between your design and the given template things as spacing, alignments, and  user experience kindly try to make your design look like the given one below 👇 are some highlighted parts to consider  
* ## YOURS 
image
image

## GIVEN DESIGN 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Amazinggracee 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Best practice followed for styling 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Amazinggracee) [reply src/App.css:338]: Hello @Davidosky007, thank you for the review. I have implemented the changes.

File: src/App.css [hunk 2]
```diff
@@ -307,20 +334,37 @@
...
```
Comment (DuaneDave) [inline src/App.css:338]: Good job working on the navlinks, they look pretty now. However, you have not worked on the alignment of the book container to start and end where the navigation starts and end. Also, the book container is still overflowing out of its parent container, consider resolving this to make the UI match the provided design.

Your design
aa


Furthermore, the widths for the input field needs to be worked on a little as pointed in the last reviewer's screenshot. The width for the book title is bigger than the others and as a result causes it parent container to take up the full width of the container where it lives. Consider resolving
title
 this and cross check your design with the template before asking for another review.

Template design

237646829-62c179de-99eb-4355-a2fa-f229053154c6

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Amazinggracee 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Navlinks inconsistency is nicely resolved


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Amazinggracee 👋 

## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**